app_dart/lib/src/service/config.dart

```diff
@@ -29,9 +29,11 @@ import 'logging.dart';
 const String kDefaultBranchName = 'master';
 
 /// Name of an example release base branch name.
+// TODO(chillers): Delete this as it's only used for tests.
```

Comment:
Delete?

Reply:
I'd prefer to leave it in. Some of these TODOs I didn't want to distract as it's already a large diff. I can follow up in a separate PR as this should be moved directly into the tests.